examples/pony/dumb_passthrough/dumb_passthrough.pony

```diff
@@ -0,0 +1,70 @@
+/*
```

Comment:
Is there a reason we have a separate passthrough app for the alosource and TCPSource?  Elsewhere, we use command line args to choose between GenSource and TCPSource.  Why not do that with the passthrough app?